app/dash_apps/reaction_viewer/index.py

```diff
@@ -286,6 +297,49 @@ def display_value(value):
             except:
                 pass
 
+            try:
+                citation = ds.data.metadata["citations"][0]["acs_citation"]
+                citation = citation.replace("</em>", "*").replace(
+                    "<em>", "*"
+                )  # Italics 1
+                citation = citation.replace("</i>", "*").replace(
+                    "<i>", "*"
+                )  # Italics 2
+                citation = citation.replace("</b>", "**").replace("<b>", "**")  # Bold
+                citation = citation.replace(", ***", "*, **").replace(
+                    "***, ", "**, *"
+                )  # HTML vs Markdown corrections
```

Comment:
We need a `format_citation` method badly :(

Reply:
Yea :/